Makefile

```diff
@@ -73,13 +77,36 @@ osx-package: $(OSX_FONT)
 	cp README.md $(OSX_PACKAGE)
 	7z a -tzip -mx=9 $(OSX_PACKAGE).zip ./$(OSX_PACKAGE)
 
+windows-package: $(WINDOWS_FONT)
+	cp $(WINDOWS_FONT) .
```

Comment:
I considered not, but it needed a .PHONY target for easy building, and it made it consistent with the other packages where the final result ended up in the top level directory.